src/main/java/sirius/tagliatelle/compiler/RawHandler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.tagliatelle.compiler;
+
+import sirius.kernel.di.std.Register;
+import sirius.tagliatelle.emitter.Emitter;
+import sirius.tagliatelle.emitter.RawEmitter;
+
+/**
+ * Handles <tt>{@literal @}raw() {}</tt>.
+ *
+ * @see RawEmitter
+ */
+@Register(classes = ExpressionHandler.class)
+public class RawHandler extends ExpressionHandler {
+
+    @Override
+    public boolean shouldProcess(Compiler compiler) {
+        return compiler.isAtText(0, "@raw()");
```

Comment:
i dont think its a good idea to check for the braces here. this restricts the usage of whitespaces that would be possible according to the process method. The JavaDoc and process is fine and matches the other handlers, just this shouldProcess check should only check for "@raw"